src/test/java/namor/template/stack/ArrayStackTest.java

```diff
@@ -103,4 +103,34 @@ public void resizeStackWhenLastCellOfArrayIsFilled() {
         assertThat(actual).isGreaterThan(firstSize);
     }
 
+    @Test
+    public void stackWorksWithString() {
```

Comment:
I thought I asked you to delete tests which checks if java works correct. Generic types work with String and Boolean you don't need to check it